### DISCUSSION TITLE
How to maximize?

### DISCUSSION THREAD

**Discussion by nitanmarcel** [Jelmerro/Vieb#389] [Q&A] [answered]
How do you maximize the window? If that's a thing ^^

**Comment by nitanmarcel** [comment, marked as answer]
I've discovered how. You double click the app icon in the status bar :)

**Comment by Jelmerro** [comment]
As you discovered, you can double click the icon, or use keyboard shortcuts. The icon serves as replacement for the native window bar, but in case you want it back, you can start Vieb with `--window-frame=true` or use environment variables to do so: `VIEB_WINDOW_FRAME=true`.
